deps/spidershim/src/v8trycatch.cc

```diff
@@ -41,7 +41,7 @@ struct TryCatch::Impl {
     // TODO: Propagate exceptions to the caller
     //       https://github.com/mozilla/spidernode/issues/51
   }
-  Isolate* Isolate() const { return isolate_; }
+  Isolate* Iso() const { return isolate_; }
```

Comment:
Can you instead use `class Isolate* Isolate() const ...` instead?  That should help fix this error without needing to rename Isolate(), which is ugly.  We prefix names with `class` like this elsewhere in the code too.